onnx/backend/test/case/node/rmsnormalization.py

```diff
@@ -0,0 +1,172 @@
+# Copyright (c) ONNX Project Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+import numpy as np
+
+import onnx
+from onnx.backend.test.case.base import Base
+from onnx.backend.test.case.node import expect
+
+
+# RMS normalization's reference implementation
+def _rms_normalization(X, W, axis=-1, epsilon=1e-5):  # type: ignore
+    X_shape = X.shape
```

Comment:
Maybe just `shape` and `rank` will reduce mixing different identifier naming styles?

Reply:
Or, `original_shape` if you want to be more explicit.